_delphi_utils_python/delphi_utils/logger.py

```diff
@@ -115,7 +112,7 @@ def add_pid(_logger, _method_name, event_dict):
         # Use a standard wrapper class for utilities like log.warning()
         wrapper_class=structlog.stdlib.BoundLogger,
         # Cache the logger
-        cache_logger_on_first_use=True,
+        cache_logger_on_first_use=is_real,
```

Comment:
Is this the main change and everything else is formatting? Any chance you could separate the formatting into a separate commit ahead of your other changes?

Reply:
other than adding `is_real` and some docs yeah